### PR TITLE
dynamic graphite env

### PR DESCRIPTION
**Pull request by ch40s3k**

### CODE REVIEW

File: alamo_worker/cli.py [hunk 1]
```diff
@@ -60,6 +63,26 @@ def send(self, result):
...
 
+    def _get_environments(self):
+        endpoint = self.config.get('maas', 'endpoint')
+        data = {}
+        try:
+            session = Session()
+            session.verify = False
+            response = session.get(endpoint)
+
+            if response.status_code != 200:  # MaaS returns 200 upon success
```
Comment (pnuckowski) [inline alamo_worker/cli.py]: `raise_for_status()`

File: alamo_worker/cli.py [hunk 2]
```diff
@@ -6,6 +6,8 @@
 import redis
 
 from rq import Queue
```
Comment (deejay1) [inline alamo_worker/cli.py]: ordering? ;)

File: alamo_worker/plugins/graphite.py [hunk 3]
```diff
@@ -33,10 +33,20 @@ def execute(self, check):
             logger.info('Graphite: {}'.format(check))
             fields = check['fields']
             result = {}
+
+            if 'prod' in check['environment']:
```
Comment (pnuckowski) [inline alamo_worker/plugins/graphite.py]: Maybe ?
```
env = 'prod' if 'prod' in check['environment'] else 'test'
url = 'http://{}:80/render'.format(self.config.env[env]['metrics_collector']['host'])
```

**Comment by deejay1** [comment]
:+1:

File: alamo_worker/cli.py [hunk 4]
```diff
@@ -60,6 +62,28 @@ def send(self, result):
...
+        endpoint = self.config.get('maas', 'endpoint')
+        data = {}
+        try:
+            session = Session()
+            session.verify = False
+            response = session.get(endpoint)
+
+            response.raise_for_status()
+
+            if response.status_code != 200:  # MaaS returns 200 upon success
```
Comment (pnuckowski) [inline alamo_worker/cli.py]: drop this and second line

File: alamo_worker/plugins/graphite.py (hunk 3 above)
Comment (deejay1) [inline alamo_worker/plugins/graphite.py]: safer would be 
```python
env = 'prod' if 'prod' in check.get('environment') else 'test'
```

**Comment by deejay1** [comment]
:+1:

**Comment by h00dy** [comment]
:+1:

**Pull request merged**